packages/core/content-releases/server/src/controllers/release.ts

```diff
@@ -88,12 +89,42 @@ const releaseController = {
       throw new errors.NotFoundError(`Release not found for id: ${id}`);
     }
 
+    const contentManagerContentTypeService = strapi
+      .plugin('content-manager')
+      .service('content-types');
+    const contentManagerComponentsService = strapi.plugin('content-manager').service('components');
+
+    const contentTypeSchemas = await contentManagerContentTypeService.findAllContentTypes();
+    const components = await contentManagerComponentsService.findAllComponents();
+
+    const mappedContentTypeSchemas = contentTypeSchemas.reduce(
+      (
+        acc: { [key: Schema.ContentType['uid']]: Schema.ContentType },
+        schema: Schema.ContentType
+      ) => {
+        acc[schema.uid] = schema;
+
+        return acc;
+      },
+      {}
+    );
+    const mappedComponents = components.reduce(
+      (acc: { [key: Schema.Component['uid']]: Schema.Component }, component: Schema.Component) => {
+        acc[component.uid] = component;
+
+        return acc;
+      },
+      {}
+    );
```

Comment:
Would it not be beneficial to have a dictionary where you could look up a contentTypes schema and components with its uid: 

```
{
 'api::test.test': {
  	schema: {...},
        component: []
  }
}
```

Reply:
This looks like it should be its own method like `getContentTypesWithComponents`, and/or we could use the function we already have that is building an object like the one described above:

https://github.com/strapi/strapi/blob/311759d2323320bcc39a3a2150ec872d5930c3e6/packages/core/content-releases/server/src/services/release.ts#L286

Would it not make sense to do all this in one place and keep all the contentType data together instead of storing some on meta and some directly in the response body?